.env.jenkins

```diff
@@ -6,7 +6,7 @@ ETHEREUM_DEV_CLUSTER=1
 EXTENSIONS=1
 FLEET=eth.beta
 GROUP_CHATS_ENABLED=1
-HARDWALLET_ENABLED=0
+HARDWALLET_ENABLED=1
```

Comment:
does this mean that it is enabled on releases now?

Reply:
@yenda  it's only for QA testing, will be disabled before merge